apisix/cli/ops.lua

```diff
@@ -806,6 +806,7 @@ local function start(env, ...)
         local ok, err, err_no = signal.kill(pid, signone)
         if ok then
             print("APISIX is running...")
```

Comment:
delete this log

Reply:
No need to print two identical logs, you can change the two places you metioned

Reply:
ok, i fix it